### PR TITLE
PIL-1211-Wallet Backup & Import Settings

### PR DESCRIPTION
**Pull request by divyaBacancy**
1) Removed the warning  Warning: Clipboard has been extracted from react-native core and will be removed in a future release.
2) changed the Toast Design

// Backup Wallet flow from settings and logout flow

https://user-images.githubusercontent.com/66996635/141935890-1c100298-d123-4f4f-9298-5dbc9bf7a2cf.mov

// Private key scenario 

https://user-images.githubusercontent.com/66996635/141936069-d96cbd2e-ee8a-45a3-b4cf-8bc38aae3ff4.mov

### CODE REVIEW

File: src/screens/RevealBackupPhrase/RevealBackupPhrase.js [hunk 1]
```diff
@@ -77,13 +77,24 @@ class RevealBackupPhrase extends React.Component<Props, State> {
     this.props.navigation.goBack(null);
   };
 
+  handleCopyToClipboard = (copiedText: string, isPrivateKey?: boolean) => {
+    Clipboard.setString(copiedText);
+    if (isPrivateKey) {
+      Toast.show({ message: t('toast.privateKeyCopiedToClipboard'), emoji: 'ok_hand', autoClose: true });
+    } else {
+      Toast.show({ message: t('toast.seedPhraseCopiedToClipboard'), emoji: 'ok_hand', autoClose: true });
+    }
```
Comment (IAmKio) [inline src/screens/RevealBackupPhrase/RevealBackupPhrase.js:86]: Maybe?

```suggestion
      Toast.show({ message: isPrivateKey ? t('toast.privateKeyCopiedToClipboard') :  t('toast.seedPhraseCopiedToClipboard'), emoji: 'ok_hand', autoClose: true });
```

File: src/utils/themes.js [hunk 2]
```diff
@@ -160,6 +162,8 @@ export const semanticDarkThemeColors = {
   swiperButtonTrack: '#520f8c',
   swiperButtonThumb: baseColors.electricViolet,
   modalHandleBar: '#4d4d4d',
+  toastBackgroundColor: '#2e673c',
```
Comment (IAmKio) [inline src/utils/themes.js:165]: Could we add this to the theme please?

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a few minor things - thank you!

File: src/screens/RevealBackupPhrase/RevealBackupPhrase.js (hunk 1 above)
Comment (divyaBacancy) [reply src/screens/RevealBackupPhrase/RevealBackupPhrase.js]: Done

File: src/utils/themes.js (hunk 2 above)
Comment (divyaBacancy) [reply src/utils/themes.js]: Done

**Review by IAmKio — APPROVED** [review]
Great work! 🙏

**Pull request merged**